src/js/components/addData/metadata/SubmissionTypeField.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+  * SubmissionTypeField.jsx
+  * Created by Daniel Boos 6/24/20
+  */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    onChange: PropTypes.func,
+    value: PropTypes.string
+};
+
+const defaultProps = {
+    onChange: null,
+    value: ''
+};
+
+export default class SubmissionTypeField extends React.Component {
+    pickedType(type) {
+        this.props.onChange(type);
+    }
+
+    render() {
+        let isTest = false;
+        let isCertifiable = false;
+
+        if (this.props.value === "test") {
+            isTest = true;
+            isCertifiable = false;
+        }
+        else if (this.props.value === "certifiable") {
+            isTest = false;
+            isCertifiable = true;
+        }
+
+        return (
+            <div>
+                <div className="row usa-da-add-data-meta-label usa-da-duration">
+                    What type of submission do you want to create?
+                </div>
+                <div className="row">
+                    <div className="col-sm-12 pos-rel text-left usa-da-submissiontype">
+                        <div className="usa-da-submissiontype-group">
+                            <input
+                                type="radio"
+                                id="usa-da-submisisontype-test"
+                                name="submisisontype"
+                                value="certifiable"
+                                onClick={this.pickedType.bind(this, 'test')}
```

Comment:
Can these binds be made in the constructor? Also, the value is flipped from the onClick